marathon_quiz.cpp

```diff
@@ -284,7 +285,22 @@ void displayQuestion(Question* q, int* questionNumber) {
         printf("%d. %s\n", i + 1, q->options[i]);
     }
 }
-
+/*
+Display question by specific category
+*/
+void displayQuestionsByCategory(Question* head, int category){
+	Question * current= head;
+	int found=0;
+	while (current != NULL){
+		if(current->category==category){
+			printf("\n%c %s\n", 254, current->question);
```

Comment:
@alfredoMoralesM  please change this line printf("\n%c %s\n", 254, current->question); to printf("\n-> %s\n", current->question);, because the character 254 in ASCII is not recognized in linux terminal.